components/TakePicture/TakePicture.js

```diff
@@ -0,0 +1,44 @@
+import React, {useState, useEffect} from 'react'
+import { StyleSheet, Text, View, Image } from 'react-native'
+import Constants from 'expo-constants'
+import * as ImagePicker from 'expo-image-picker'
+import * as Haptics from 'expo-haptics';
+import {styles} from '../PickImage/PickImageStyles'
+
+import { Button } from "react-native-paper"
+
+export default function TakePicture() {
+    const [localUri, setUri] = useState("")
+
+  let openCameraAsync = async () => {
```

Comment:
So cool to see yall taping into the camera, the haptics, and all of the other mobile specific options here. I Hope Native is treating you well!